pkg/runtime/openshift.go

```diff
@@ -1,5 +1,134 @@
 package runtime
 
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/openshift/ansible-service-broker/pkg/clients"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
 func (o openshift) getRuntime() string {
 	return "openshift"
 }
+
+func (o openshift) shouldJoinNetworks() (bool, PostSandboxCreate, PostSandboxDestroy) {
+	ocli, err := clients.Openshift()
+	if err != nil {
+		log.Errorf("unable to get openshift client - %v", err)
+		// Defaulting if anything goes wrong to not join the networks.
+		return false, nil, nil
+	}
+	pluginName, err := ocli.GetClusterNetworkPlugin()
+	log.Debugf("plugin for the network - %v", pluginName)
+	if err != nil {
+		// The plugins could not be defined. ex: when using oc cluster up
+		// or a pure k8s cluster. Therefore making this a notice.
+		log.Noticef("unable to retrieve the network plugin, defaulting to not joining networks - %v", err)
+		// Defaulting to not join the networks.
+		return false, nil, nil
+	}
+
+	// Case insensitive check here because want to prepare if things change.
+	if strings.ToLower(pluginName) == "redhat/openshift-ovs-multitenant" {
+		log.Debugf("stating that the pluginname is multitenant - %v", pluginName)
+		return true, addPodNetworks, isolatePodNetworks
```

Comment:
as mentioned above, there's really no need to re-isolate the transient network on cleanup